src/site/content/en/blog/compress-images-avif/index.md

```diff
@@ -0,0 +1,348 @@
+You have successfully built avifenc!
+
+### Understanding the avifenc command line parameters
+
+avifenc uses the command-line structure:
+
+```shell
+./avifenc [options] input.file output.avif
+```
+
+The basic parameters for avifenc used in this tutorial are:
+
+<table class="w-table">
+  <thead>
+    <tr>
+      <th>avifenc</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>--min 0</td><td>Set min quantizer for color to 0</td>
+    </tr>
+    <tr>
+      <td>max 63</td><td>Set max quantizer for color to 63</td>
+    </tr>
+    <tr>
+      <td>--minalpha 0</td><td>Set min quantizer for alpha to 0</td>
+    </tr>
+    <tr>
+      <td>--maxalpha 63</td><td>Set max quantizer for alpha to 63</td>
+    </tr>
+    <tr>
+      <td>-a end-usage=q</td><td>Set the rate control mode to Constant Quality (Q) mode</td>
+    </tr>
+    <tr>
+      <td>-a cq-level=Q</td><td>Set quantize level for both color and alpha to Q</td>
+    </tr>
+    <tr>
+      <td>-a color:cq-level=Q</td><td>Set quantize level for color to Q</td>
+    </tr>
+    <tr>
+      <td>-a alpha:cq-level=Q</td><td>Set quantize level for alpha to Q</td>
+    </tr>
+    <tr>
+      <td>-a tune=ssim</td><td>Tune for SSIM (default is to tune for PSNR)</td>
+    </tr>
+    <tr>
+      <td>--jobs J</td><td>Use J worker threads (default: 1)</td>
+    </tr>
+    <tr>
+      <td>--speed S</td><td>Set encoder speed from 0-10 (slowest-fastest. default: 6)</td>
+    </tr>
+  </tbody>
+</table>
+
+The cq-level option sets the quantize level (0.63) to control the quality for color or alpha.
+
+{% Aside %}
+You can think of cq-level is the "amount" of quantization, so a lower value yields higher quality
+{% endAside %}
+
+{% Aside %}
+Higher speed settings will run faster,
+but produce worse compression efficiency and quality.
+This tutorial uses the default setting of Speed 6,
+which we recommend as a balance of encode speed, compression efficiency, and quality.
+{% endAside%}
+
+### Create an AVIF image with default settings
+
+The most basic parameters for avifenc to run, are setting the input and output files.
+
+```shell
+./avifenc happy_dog.jpg happy_dog.avif
+```
+
+We recommend the following command line to encode an image, say at quality level 18:
+
+```shell
+./avifenc --min 0 --max 63 -a end-usage=q -a cq-level=18 -a tune=ssim happy_dog.jpg happy_dog.avif
+```
+
+{% Aside %}
+<code>"--min 0 --max 63 -a end-usage=q -a cq-level=18 -a tune=ssim"</code>
+are the recommended settings for AVIF images.
+If the image has an alpha channel,
+add <code>"--minalpha 0 --maxalpha 63"</code>.
+To specify different quantize levels for color and alpha,
+replace <code>"-a cq-level=18"</code> with, say,
+<code>"-a color:cq-level=18 -a alpha:cq-level=10"</code>.
+{% endAside %}
+
+
+Avifenc has a lot of options that will affect both quality and speed.
+If you want to see the options and learn more about them just run <code>./avifenc</code>
+
+You now have your very own second AVIF image!
```

Comment:
Nit: It seems that we should remove the word "second", or perhaps change it to "first".